src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -410,7 +410,10 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 self.copy_op(self.operand_index(args[0], index)?, dest)?;
             }
             // FIXME(#73156): Handle source code coverage in const eval
-            sym::count_code_region => (),
+            sym::count_code_region
+            | sym::coverage_counter_add
+            | sym::coverage_counter_subtract
+            | sym::coverage_unreachable => (),
```

Comment:
Heads up @rust-lang/wg-const-eval, this PR is adding some new stub intrinsics which are used by the [LLVM code coverage feature](https://github.com/rust-lang/compiler-team/issues/278) @richkadel is adding.

Reply:
Also cc @rust-lang/lang. This PR adds a few new intrinsics which are being used by the [LLVM code coverage feature](https://github.com/rust-lang/compiler-team/issues/278) @richkadel is adding. Currently, these intrinsics are no-ops when evaluated at compile time (see https://github.com/rust-lang/rust/issues/73156 for possible future extensions where these could measure code coverage when run by the const eval machinery).

Reply:
These intrinsics seem like they're an implementation detail at this point, is that correct?

Reply:
I think that's a fair characterization. The intrinsics are injected via a MIR pass enabled by the `-Zinstrument-coverage` flag.

Reply:
OK 👍 then I think it's all fine from the lang team perspective.

Reply:
Thanks Niko!